recipes/libcurl/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(CURL REQUIRED)
+
+add_executable(${PROJECT_NAME} test_package.c)
```

```suggestion
add_executable(${PROJECT_NAME} ..test_package/test_package.c)
```
Then, you can remove test_v1_package/test_package.c